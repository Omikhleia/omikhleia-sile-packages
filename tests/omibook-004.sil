\begin[class=omibook]{document}
\nofolios
\noheaders
% Test case from the omibook customization.
% Can I remove that dot after the section numbers?
% (Or change it)
\style:redefine[name=sectioning:other:number, as=_1, inherit=true]{
    \numbering[after=.]}%
\section{One}

Section should have a dot after its number.

\style:redefine[name=sectioning:other:number, as=_2, inherit=true]{
    \numbering[after=false]}%
\section{Two}

Section should not have the dot any longer.

\style:redefine[name=sectioning:other:number, as=_3, inherit=true]{
    \numbering[after=–, before=§, kern=3spc]}%
\section{Two}

Section should have an emdash after its number, a § before it, and larger spacing.

\end{document}
