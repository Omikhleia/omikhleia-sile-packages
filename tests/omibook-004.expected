Set paper size 	595.275597	841.8897729
Begin page
Mx 	59.5276
My 	105.2178
Set font 	Gentium Plus;12;800;;normal;;LTR
T	20 w=5.6309	(1)
Mx 	65.1584
T	17 w=2.7480	(.)
Mx 	71.1346
T	50 81 72 w=19.3652	(One)
Mx 	59.5276
My 	123.2178
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 72 70 87 76 82 81 w=30.4492	(Section)
Mx 	92.6505
T	86 75 82 88 79 71 w=27.6172	(should)
Mx 	122.9415
T	75 68 89 72 w=19.6582	(have)
Mx 	145.2735
T	68 w=4.5898	(a)
Mx 	152.5371
T	71 82 87 w=13.6719	(dot)
Mx 	168.8827
T	68 73 87 72 85 w=19.7217	(after)
Mx 	191.2782
T	76 87 86 w=10.0146	(its)
Mx 	203.9666
T	81 88 80 69 72 85 w=32.5049	(number)
Mx 	236.4715
T	17 w=2.2900	(.)
Mx 	59.5276
My 	147.2178
Set font 	Gentium Plus;12;800;;normal;;LTR
T	21 w=5.6309	(2)
Mx 	68.3867
T	55 90 82 w=21.1172	(Two)
Mx 	59.5276
My 	165.2178
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 72 70 87 76 82 81 w=30.4492	(Section)
Mx 	92.6503
T	86 75 82 88 79 71 w=27.6172	(should)
Mx 	122.9410
T	81 82 87 w=13.9893	(not)
Mx 	139.6037
T	75 68 89 72 w=19.6582	(have)
Mx 	161.9354
T	87 75 72 w=13.5791	(the)
Mx 	178.1880
T	71 82 87 w=13.6719	(dot)
Mx 	194.5334
T	68 81 92 w=15.0391	(any)
Mx 	212.2459
T	79 82 81 74 72 85 w=26.7139	(longer)
Mx 	238.9598
T	17 w=2.2900	(.)
Mx 	59.5276
My 	189.2178
Set font 	Gentium Plus;12;800;;normal;;LTR
T	134 w=5.6309	(§)
Mx 	65.1584
T	22 w=5.6309	(3)
Mx 	70.7893
T	178 w=5.6309	(–)
Mx 	84.3479
T	55 90 82 w=21.1172	(Two)
Mx 	59.5276
My 	207.2178
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 72 70 87 76 82 81 w=30.4492	(Section)
Mx 	92.6330
T	86 75 82 88 79 71 w=27.6172	(should)
Mx 	122.9063
T	75 68 89 72 w=19.6582	(have)
Mx 	145.2207
T	68 81 w=10.1074	(an)
Mx 	157.9843
T	72 80 71 68 86 75 w=31.8213	(emdash)
Mx 	192.4617
T	68 73 87 72 85 w=19.7217	(after)
Mx 	214.8396
T	76 87 86 w=10.0146	(its)
Mx 	227.5104
T	81 88 80 69 72 85 w=32.5049	(number)
Mx 	260.0153
T	15 w=2.2900	(,)
Mx 	264.9615
T	68 w=4.5898	(a)
Mx 	272.2075
T	134 w=4.6924	(§)
Mx 	279.5561
T	69 72 73 82 85 72 w=26.4160	(before)
Mx 	308.6283
T	76 87 w=6.1523	(it)
Mx 	314.7806
T	15 w=2.2900	(,)
Mx 	319.7268
T	68 81 71 w=15.3076	(and)
Mx 	337.6906
T	79 68 85 74 72 85 w=24.7168	(larger)
Mx 	365.0636
T	86 83 68 70 76 81 74 w=31.2256	(spacing)
Mx 	396.2892
T	17 w=2.2900	(.)
End page
Finish
